docs/source/getting-started/setting-up-virtualization-environment.adoc

```diff
@@ -111,6 +111,35 @@ $ sudo sed -r 's/group=".+"/group="kvm"/1' /etc/libvirt/qemu.conf > /etc/libvirt
 $ newgrp libvirt
 ----
 
+[[kvm-driver-networking]]
+=== Configure libvirt networking
+
+On some distributions, the *default* libvirt network will already be started upon installation.
+On others, you'll have to start it manually.
```

Comment:
Feels like repetition... 'on'... 'on'... "Some distributions setup the default network for you, while on others this might have to be done manually. Below you will find instructions to define the network." or somethimg similar.